pkg/scheduler/framework/plugins/defaultpodtopologyspread/default_pod_topology_spread.go

```diff
@@ -86,3 +94,135 @@ func New(_ *runtime.Unknown, handle framework.FrameworkHandle) (framework.Plugin
 		calculateSpreadPriorityReduce: calculateSpreadPriorityReduce,
 	}, nil
 }
+
+// SelectorSpread contains information to calculate selector spread priority.
+type SelectorSpread struct {
+	serviceLister     corelisters.ServiceLister
+	controllerLister  corelisters.ReplicationControllerLister
+	replicaSetLister  appslisters.ReplicaSetLister
+	statefulSetLister appslisters.StatefulSetLister
+}
+
+// NewSelectorSpreadPriority creates a SelectorSpread.
+func NewSelectorSpreadPriority(
+	serviceLister corelisters.ServiceLister,
+	controllerLister corelisters.ReplicationControllerLister,
+	replicaSetLister appslisters.ReplicaSetLister,
+	statefulSetLister appslisters.StatefulSetLister) (priorities.PriorityMapFunction, priorities.PriorityReduceFunction) {
+	selectorSpread := &SelectorSpread{
+		serviceLister:     serviceLister,
+		controllerLister:  controllerLister,
+		replicaSetLister:  replicaSetLister,
+		statefulSetLister: statefulSetLister,
+	}
+	return selectorSpread.CalculateSpreadPriorityMap, selectorSpread.CalculateSpreadPriorityReduce
```

Comment:
We don't need this type and function, we already have ```DefaultPodTopologySpread``` defined in line 36 and ```New``` at line 82